Makefile

```diff
@@ -150,7 +154,7 @@ bin/kind: test/k8s/Dockerfile
 
 # Use a static build of the shims for better compatibility.
 # Using cross defaults to x86_64-unknown-linux-musl, which creates a static build.
-test/k8s/_out/img-%: CARGO=cross
+test/k8s/_out/img-%: CARGO=cross TARGET=$(shell uname -m)-unknown-linux-musl
```

Comment:
**praise:** Good catch! Thanks for fixing it!

Reply:
Don't love that it's here and above but tried a few different ways to try this but seem to be the only way I could get Make to overload that value, since it saw empty as a set value and couldn't unset it here.

Reply:
What if you set `TARGET=` here? (i.e., an empty value)
I **think** the `TARGET ?= ...` from above should override it if its empty.

Reply:
Ok, no, `?=` doesn't override empty values.
Stackoverflow suggests replacing with
```makefile
TARGET = $(or $(TARGET),$(shell uname -m)-unknown-linux-musl)
```

Reply:
I tried that and it same it as Empty value not an `unset` value, and so `?=` didn't take the default and used "" instead

Reply:
using different approach